### ISSUE TITLE
can't find cloudius/osv-base

### ISSUE THREAD

**Issue by photoszzt** [cloudius-systems/capstan#180]
I'm trying to build perl in apps repo with capstan, I get the following error:

remote: Repository not found.
fatal: repository 'https://github.com/cloudius/osv-base/' not found

**Comment by wkozaczuk** [comment]
Yes `build-capstan-mpm-packages` can be used to build any app from apps as a package.

I fully agree that we need to make using capstan less confusing. I think better documentation is key. Getting rid of some files - sure. 

But I think there is nothing wrong in having these 2 options:
- base-image + Capstan file way
- package way

They are both inspired by Docker's way of creating images. Each has some merits. I was hoping this paragraph from the revised Capstan readme will clarify it (work in progress) - https://github.com/wkozaczuk/capstan/tree/revise_docs:

"Capstan has been designed to operate in two distinct modes:
- a mode based on Capstanfile (conceptually similar to Dockerfile) where the Capstanfile specifies base OSv image, optional build command, list of files to append to the image and the boot command line (for details look at Capstanfile specification)
- a mode based on packages (conceptually similar to Docker compose), where Capstan uses prebuilt artifacts: precompiled OSv kernel, Java runtime from the Linux host, precompiled MySQL, and many more."
